docs/wiki/guides/current/DSHOT-RPM-Filtering.md

```diff
@@ -40,101 +40,29 @@ If the RPM Filter is enabled but one or more of the ESC's are not supplying vali
-
-#### BlueJay
-
-BlueJay is a more recent RPM-aware BlHeli-S firmware. Orginally developed by Mathias, a Betaflight developer, more recently BlueJay has been transferred to the BirdSanctuary team and is maintained by Damosvil and Stylesuxx in close partnership with the esc-configurator project. BlueJay is easily flashed with an elegant [online flashing tool](https://esc-configurator.com).  Custom offline configurator (https://github.com/mathiasvr/bluejay-configurator/releases) is available but not recommended for current releases. The firmware supports both L and H type ESCs as well as newer Z type, with a range of options, and has been tested on various ESC models. [Extended Dshot Telementry (EDT)](https://github.com/bird-sanctuary/extended-dshot-telemetry) was created by the BlueJay team and allows ESCs without a separate telemetry UART to send additional telemetry alongside RPM data. EDT enables BlueJay ESCs to report voltage, current and temperature as well as signalling error events.
-
-| Version    | Recommended | Comment                                                                                                    |
-| ---------- | ----------- | ---------------------------------------------------------------------------------------------------------- |
-| 0.19.2     | `Y`         | Current stable version, supports RPM filtering and EDT. Upgrade to this version unless you fly 3D mode     |
-| 0.20       | `N`         | Old testing version, withdrawn from release due to problems.                                               |
-| 0.20.1-RC2 | `N`*        | Latest test code. Includes EDT support. *Recommended if you fly 3D as this fixes 3D mode                   |
+Your ESC must support Dshot, and be running [a suitable firmware](/docs/wiki/getting-started/hardware/esc) to support RPM filtering. At the minimum RPM telemetry is required, so the ESC must support bidirectional Dshot. As EDT extends bidirectional Dshot any ESC supporting EDT also support RPM telemetry.
 
 ## Betaflight Configuration
 
 ### Motor Poles
 
 When running 8k8k, choose Dshot600. The ESCs report eRPM. This must be converted to RPM using the number of magnets of the motors. The magnets to count are those on the bell of the motor. Do not count the stators where the windings are located. Typical 5" motors have 14 magnets, so that is the default setting. Smaller motors have fewer magnets, often 12. Count them or look up the motor specs. If you don't have 14 magnets, change the number of magnets using Betaflight Configurator on the Configuration tab.
 
-### DShot150, DShot300 or DShot600?
+### Dshot150, Dshot300 or Dshot600?
 
 For 4k PID loops, eg 8k4k or 4k4k, use Dshot 300 for greatest reliability; Dshot 600 may be used but is not recommended.
```

Comment:
maybe
For 4k PID loops, eg 8k4k or 4k4k, or 3.2k PID loops, use Dshot 300 for greatest reliability; Dshot 600 may be used but provides no advantage and is more susceptible to external noise.